### PR TITLE
[tts] refactor parakeet example

### PR DESCRIPTION
**Pull request by yt605155624**
The hard link in readme hasn't change here.
The benchmark and chain hasn't fix here.
All parakeet example has test alreadly.

### CODE REVIEW

File: examples/aishell3/tts3/run.sh [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+gpus=0,1
+stage=0
+stop_stage=100
+
+
+conf_path=conf/default.yaml
+train_output_path=exp/default
+ckpt_name=snapshot_iter_482.pdz
+
```
Comment (zh794390558) [inline examples/aishell3/tts3/run.sh:14]: source ${MAIN_ROOT}/utils/parse_options.sh
其他类同

Comment (yt605155624) [reply examples/aishell3/tts3/run.sh:14]: done

**Review by zh794390558 — DISMISSED** [review]
LGTM

**Review by zh794390558 — APPROVED** [review]
LGTM

**Pull request merged**